allennlp/models/decomposable_attention.py

```diff
@@ -116,11 +116,13 @@ def forward(self,  # type: ignore
         embedded_hypothesis = self._text_field_embedder(hypothesis)
         premise_mask = get_text_field_mask(premise).float()
         hypothesis_mask = get_text_field_mask(hypothesis).float()
+        premise_sequence_lengths = get_lengths_from_binary_sequence_mask(premise_mask)
+        hypothesis_sequence_lengths = get_lengths_from_binary_sequence_mask(hypothesis_mask)
 
         if self._premise_encoder:
-            embedded_premise = self._premise_encoder(embedded_premise)
+            embedded_premise = self._premise_encoder(embedded_premise, premise_sequence_lengths)
         if self._hypothesis_encoder:
-            embedded_hypothesis = self._hypothesis_encoder(embedded_hypothesis)
+            embedded_hypothesis = self._hypothesis_encoder(embedded_hypothesis, hypothesis_sequence_lengths)
```

Comment:
Also, I think there's at least one encoder that was missed - the `TokenCharactersEmbedder` has an encoder that needs to be updated to use this.